### ISSUE TITLE
MathQuill and Slack

### ISSUE THREAD

**Issue by soegaard** [mathquill/mathquill#1033]
1. The community section [1] does not list Slack.

2. The first paragraph of the docs at [2] mentions Slack and links to 
        http://slackin.mathquill.com/
    But it seems that link is dead?
     

[1] http://mathquill.com/#community
[2] http://docs.mathquill.com/en/latest/

**Comment by nedredmond** [comment]
Yeah, the Slack link is a defunct relic of permanent invite links. Those no longer work across the board. I took them off the `README.md` but missed the docs. Feel free to send a PR like I did!

**Comment by soegaard** [comment]
Ping.

If both irc and Slack are dead, is there an alternative forum or mailing list for MathQuill?

**Comment by nedredmond** [comment]
I haven't been to the Slack, but I don't believe there is an active place for discussions. The easiest lift would be activating the Discussions feature on this repo, but those with the ability to do so are otherwise engaged.
